docs/faq/proof-of-stake/voting-service-providers.md

```diff
@@ -59,6 +59,12 @@ In the unlikely scenario your VSP goes down permanently, you can still vote your
 
 ---
 
+#### 9. My VSP went down and I lost the redeem script, is there a way to recover it?
+
+When a ticket votes or is revoked, that redeem script is revealed as a part of the process. Redeem scripts that have been used at least once to vote or revoke a ticket can be [recovered using Decred's block explorer](/proof-of-stake/redeem-script/#recovery-methods).
```

Comment:
Internal links need to be relative paths to `.md` files in order to benefit from of the link validation built into mkdocs.

In this case we want to use `../../proof-of-stake/redeem-script.md#recovery-methods` (also note removal of the final forward slash